_posts/2021-05-04-how-to-update-your-ref-arch.adoc

```diff
@@ -439,77 +464,69 @@ that are not covered by the automated scripts. If you wish to update further, fi
 the migration guides in the release notes of the Service Catalog to bump beyond that version.
 ====
 
-    * Convert each service listed in `dependencies` to a `dependency` block. Use <<service_migration_guides,the dedicated
-      service migration guide>> as a reference for what dependency blocks are needed.
-    * Update the inputs to adapt to the Service Catalog Terraform module, using the dedicated service migration guide listed
-      below. You can also use `terragrunt validate-inputs` as a sanity check.
+    * Find the <<service_migration_guides,dedicated service migration guide>> for the service.
+    * Using that guide, update the inputs to adapt to the Service Catalog Terraform module.
+        * You can use `terragrunt validate-inputs` as a sanity check.
+        * Remove the `dependencies` block, if any.
+        * Use `dependency` blocks. Use <<service_migration_guides,the dedicated
+          service migration guide>> as a reference for what dependency blocks are needed.
     * Add new required inputs, using `dependency` references as needed.
     * Remove or rename unused variables.
+    * Ensure you include inputs for backward compatibility mentioned in the dedicated guide!
 
-. Run `terragrunt state list` to sanity check the state backend configuration. You are watching for the following:
+. Run `terragrunt state list` to sanity check the state backend configuration. Watch for the following:
     * You should NOT get any prompts from Terragrunt to create a new S3 state bucket. If you get the prompt, this means that
       either you are authenticating to the wrong account, or that the bucket name was misconfigured in the root
       `terragrunt_service_catalog.hcl` file.
     * You should see resources listed in the state. If the command returns nothing, that means you are not properly linked
-      to the old state file.  Double check the `key` attribute of the `remote_state` block in the root
+      to the old state file. Double check the `key` attribute of the `remote_state` block in the root
       `terragrunt_service_catalog.hcl` config.
```

Comment:
@yorinasub17 I have not personally checked whether it works to run `terragrunt state list` after having made changes to the terragrunt.hcl. Also note, previously, the state file backup have happened _after_ this step.